golang/cosmos/x/vtransfer/keeper/keeper.go

```diff
@@ -101,12 +102,42 @@ func (k Keeper) GetReceiverImpl() vibctypes.ReceiverImpl {
 	return k
 }
 
+// Extract the base address from the packet sender (if senderIsLocal) or
+// receiver (if !senderIsLocal), since the local ibcModule doesn't understand
+// address parameters.
+func (k Keeper) packetWithOnlyBaseAddresses(packet channeltypes.Packet, senderIsLocal bool) channeltypes.Packet {
```

Comment:
_question_

Could we add some unit tests for this function? If nothing else, just to use unit tests to document the behavior of this function

Reply:
Definitely!  I'll work on those.